inference/inference.py

```diff
@@ -321,9 +321,8 @@ def autoregressive_inference(params, ic, valid_data_full, model):
               **grad_mag_target_metrics
             }
           )
-              
 
-    if params.log_to_wandb:
+    if params.log_to_wandb and hasattr(params, 'epoch'):
```

Comment:
What if a user was just calling the `inference.py` script but still wanted to output these 5-day and 10-day metrics? Don't think we should require `epoch` to be in `params` here.

Reply:
Right

Reply:
Is the `hasattr(params, 'epoch')` still here intentionally? Don't think we need it?